apps/token/src/routes/staking/node-list.tsx

```diff
@@ -251,11 +260,13 @@ export const NodeList = ({ epoch }: NodeListProps) => {
   return (
     <AsyncRenderer loading={loading} error={error} data={nodes}>
       {epoch && epoch.timestamps.start && epoch.timestamps.expiry && (
-        <EpochCountdown
-          id={epoch.id}
-          startDate={new Date(epoch.timestamps.start)}
-          endDate={new Date(epoch.timestamps.expiry)}
-        />
+        <div className="mb-20">
```

Comment:
question (non-blocking): could we move this to the root div of the epoch countdown? Does this look reasonable in other places it is rendered?

Reply:
Where it's used elsewhere it's in its own page section with a bit more space around it, but here I don't want quite as much distance, because the epoch data and the validators involved in the epoch are coupled together in a page section, whereas elsewhere the epoch countdown is separated in its own section. I think it's appropriate to have custom spacing with this wrapper div in this instance.